lib/triannon-client/triannon_client.rb

```diff
@@ -38,15 +34,15 @@ def delete_annotation(id)
       check_id(id)
       begin
         response = @site["/annotations/#{id}"].delete
-        # HTTP DELETE response codes:
-        # A successful response SHOULD be 200 (OK) if the response includes an
-        # entity describing the status, 202 (Accepted) if the action has not yet
-        # been enacted, or 204 (No Content) if the action has been enacted but the
-        # response does not include an entity.
+        # HTTP DELETE response codes: A successful response SHOULD be
+        # 200 (OK) if the response includes an entity describing the status,
+        # 202 (Accepted) if the action has not yet been enacted, or
+        # 204 (No Content) if the action has been enacted but the response
+        # does not include an entity.
```

Comment:
um ... looks like you sort of made this up from looking at HTTP spec;  it's not what the client does (per specs I got from Rob).  For Delete, it's 204 or bust:  #https://github.com/sul-dlss/triannon/blob/master/app/controllers/triannon/annotations_controller.rb#L127-153